SKILL.tsv

```diff
@@ -3719,45 +3719,45 @@ SKILL_20150414_003718	Order summoned demon to move and attack in target area.
 SKILL_20150414_003726	{memo X}Servant HP #{CaptionRatio}#
 SKILL_20150414_003727	{memo X}Control time and temporarily increase attack speed of yourself or 1 ally in front.
 SKILL_20150414_003728	{memo X}Control time and make the monster appear again where it was killed.
 SKILL_20150414_003729	Stop time of monsters within target area. Enemies do not receive damage when time is stopped.
-SKILL_20150414_003730	Control time and decrease movement speed of enemy.
+SKILL_20150414_003730	Control time and decrease the movement speed of enemies.
 SKILL_20150414_003731	{memo X}Control time and increase movement speed of yourself or 1 ally in front.
-SKILL_20150414_003732	Control the time around and temporarily change it back to previous state.
-SKILL_20150414_003733	{#993399}{ol}[Magic]{/}{/}{nl}Collect enemy corpse.
+SKILL_20150414_003732	Control time in an area, temporarily changing it back to its previous state.
+SKILL_20150414_003733	{#993399}{ol}[Magic]{/}{/}{nl}Collect an enemy's corpse.
 SKILL_20150414_003734	{memo X}Attack #{SkillAtkAdd}#{nl}Collect corpse of each type
 SKILL_20150414_003735	{memo X}Make Shogos molded after the card inserted in Necronomicon. Necronomicon can equip only Animal, Plant, and Mutant type cards.
 SKILL_20150414_003736	{memo X}Use #{CaptionRatio}# each of {nl}Plant, Beast, Mutant corpse
 SKILL_20150414_003737	{#993399}{ol}[Magic]{/}{/}{nl}Throw corpse parts to target area and attack nearby enemies.
 SKILL_20150414_003738	{memo X}Attack #{SkillAtkAdd}#{nl}Spalsh #{SkillSR}#{nl}Use #{CaptionRatio}# each of{nl}Plant, Beast. Mutant monster corpse
-SKILL_20150414_003739	{#993399}{ol}[Magic]{/}{/}{nl}Cover around with corpse parts and attack enemies nearby.
+SKILL_20150414_003739	{#993399}{ol}[Magic]{/}{/}{nl}Encircle yourself with corpse parts that will damage nearby enemies.
 SKILL_20150414_003740	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 15 seconds{nl}Use #{CaptionRatio}# Mutant corpse
 SKILL_20150414_003741	Make a wall of corpse and block enemy attacks. Enemies attack neaby the wall becomes decomposed.
 SKILL_20150414_003742	{memo X}Dirty Wall #{CaptionRatio}#{nl}Dirty Wall Duration15 seconds{nl}{#339999}{ol}[Decompose]{/}{/} Duration #{CaptionTime}# seconds{nl}{nl}Use #{CaptionRatio2}# each of {nl}Plant, Beast, Mutant corpse
-SKILL_20150414_003743	Collect piece of corpse left where the enemy died.
-SKILL_20150414_003744	{#993399}{ol}[Magic]{/}{/}{nl}Explode your item on the ground to attack enemies.
+SKILL_20150414_003743	Collect the corpse left over from where an enemy died.
+SKILL_20150414_003744	{#993399}{ol}[Magic]{/}{/}{nl}Explode an item on the ground to attack enemies.
 SKILL_20150414_003745	{memo X}Dig the ground and get ores for alchemy.
-SKILL_20150414_003746	Open shop to roast and enhance gem. Penalty of roasted gem decreases.
+SKILL_20150414_003746	Open shop to roast and enhance gems. The penalty of a roasted gem is decreased.
```

Comment:
Depends on the level.  It only removes the negative effects up to a certain level, based on the level of the user.